### PR TITLE
Miscellaneous Settings PRIM-266

### PR DESCRIPTION
**Pull request by JPPrimus123**

### CODE REVIEW

File: PageObjects/BankSettingPageObject.cs [hunk 1]
```diff
@@ -58,9 +58,35 @@ public class BankSettingPageObject : DriverHelper
...
+        public By homepagelink => ByLocator("//a[contains(text(),'Home')]");
+        public By search => ByLocator("//div//input[@data-placeholder='Search Here']");
+        public By resultdata => ByLocator("//tbody/tr/td[contains(text(),'Jay')]");
         public BankSettingPageObject(IWebDriver driver) : base(driver)
         {
         }
+        public void GetResult()
+        {
+            Click(resultdata);
+        }
```
Comment (ahaynes91) [inline PageObjects/BankSettingPageObject.cs:72]: This is selecting the application but where is the validation that the DTI threshold component exists (or doesn't exist) on the SBO?

**Review by ahaynes91 — CHANGES_REQUESTED** [review]
Need to add a validation step that checks for the component's existence.

File: PageObjects/BankSettingPageObject.cs (hunk 1 above)
Comment (ahaynes91) [reply PageObjects/BankSettingPageObject.cs:72]: What I am asking for is after clicking to open the application, verify on the Solution Based Outcome tab the DTI Threshold component's existence.

**Review by ahaynes91 — CHANGES_REQUESTED** [review]

**Review by ahaynes91 — APPROVED** [review]

**Comment by JPPrimus123** [comment]
hi Alex,

Thanks

Regards
Jay Prakash
________________________________

**Pull request merged**